app/configurations/config.matka.js

```diff
@@ -524,4 +524,6 @@ export default {
       virtualMonitorBaseUrl: 'https://matkamonitori.digitransit.fi/',
     },
   },
+  // Notice! Turning on this setting forces the search for car routes.
```

Comment:
Same as above.

Reply:
Also, since the kela configuration is now built on top of the matka configuration (which was a mistake we should fix at some point), set this to false in the kela configuration.

Reply:
Added a clarification for matka and set this option to false for kela.